File: rails-demo/app/controllers/pages/messages_v2_controller.rb
class Pages::MessagesV2Controller < ApplicationController
  def compose
    @title = 'Compose message v2 - Pages'
  end

  def item
    @title = 'Message item v2 - Pages'
  end

  def list
    @title = 'Messages list v2 - Pages'
  end

end

File: laravel-demo/app/Http/Controllers/Pages/MessagesV2Controller.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class MessagesV2Controller extends Controller
{
    public function compose()
    {
        return view('pages.messages-v2.compose', ['title' => 'Compose message v2 - Pages']);
    }

    public function item()
    {
        return view('pages.messages-v2.item', ['title' => 'Message item v2 - Pages']);
    }

    public function list()
    {
        return view('pages.messages-v2.list', ['title' => 'Messages list v2 - Pages']);
    }

}
